### PR TITLE
Priorities: Added SCSS file to fix issues related to overly-wide linked areas.

### LINKED ISSUE #894: GoC initiatives and activities links using super wide linked area in small view
The title pretty much says it all. All versions of GCWeb seem to be affected.

The blue highlighting in the following screenshot represents the link's "clickable" area.

**Screenshot:**
goc-activities-initiatives_linked-area

**Example link:**
http://wet-boew.github.io/themes-dist/GCWeb/index-en.html

Was this intentionally done to make it easier for touch screen users to click the links in question? Seems kind of excessive IMO.

### PR DESCRIPTION
**Pull request by EricDunsworth**
Fixes #894.

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
Good job for finding a fix but shouldn't that be limited to the view causing problems instead of applying it to all views?

**Comment by EricDunsworth** [comment]
@LaurentGoderre I was originally only targeting small view, but while testing I noticed the same issue was occurring in medium and large views and figured I might as well make it work in all screen views.

Anyway, I've tested it in FF, IE8, and Opera and they all seem fine. I also have ``:focus`` outlines working identically to before.

I've just noticed one issue, though...  ``:active`` outlines don't show up. ``:active`` isn't covered in the new SCSS file, but when I tried sneaking it in with the same inherit trick that worked for ``:focus``, nothing appeared. It is possible to define explicit outlines to get that working. Thoughts?

**Comment by EricDunsworth** [comment]
Hmm, seems like my previous post's issue is being triggered by an unrelated CSS conflict. And browser-default outlines might not be in use on those links in the first place. Maybe I'll be able to fix it to my liking after all.

**Comment by LaurentGoderre** [comment]
Let us know!

**Comment by EricDunsworth** [comment]
Fixed :)! Should be good to go now.

**Comment by EricDunsworth** [comment]
@LaurentGoderre Is something wrong with Travis-CI atm? Seems like it's stalled atm.

**Comment by LaurentGoderre** [comment]
It was interupted but the status change was not reported to GitHub...strange...

I restarted the build

**Comment by EricDunsworth** [comment]
Looks like it failed this time, but from what I can tell it didn't complain about anything particular to this PR. Not seeing any linting errors related to my SCSS file or the like.

File: src/priorities/_screen.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/*
+ * Priorities (Government of Canada activities and initiatives) (all screen views)
+ */
+
+.gc-nttvs {
+	a {
+		&:active,
+		&:focus,  {
```
Comment (LaurentGoderre) [inline src/priorities/_screen.scss]: I am not sure there should be a comma `,` here, it might be complaining about this

**Comment by EricDunsworth** [comment]
@LaurentGoderre Wow, good catch! I completely missed that previously.

Just updated the PR... hopefully it'll work this time :).

**Comment by EricDunsworth** [comment]
Should I implement RTL support too?

AFAIK multilingual versions of GCWeb don't exist yet, so I'm not sure if adding in extra RTL SCSS would just be a waste of space.

**Comment by EricDunsworth** [comment]
Btw this PR should be good to go since my last comment's edit.

It's currently working perfectly in every browser I tested in (IE8/FF/Opera) and has full RTL support.

**Comment by shawnthompson** [comment]
@EricDunsworth happy now?!?!?

**Comment by EricDunsworth** [comment]
@shawnthompson Yep :). Thanks!

**Pull request merged**